tests/test_trial.py

```diff
@@ -165,8 +192,8 @@ def test_fixed_trial_suggest_uniform():
 def test_fixed_trial_suggest_loguniform():
     # type: () -> None
 
-    trial = FixedTrial({'x': 1.})
-    assert trial.suggest_loguniform('x', 0., 1.) == 1.
+    trial = FixedTrial({'x': 0.99})
+    assert trial.suggest_loguniform('x', 0., 1.) == 0.99
```

Comment:
Thank you for fixing the test case.
`{'x': 1}` is not an appropriate value for this test case because `suggest_loguniform` does not include `high`.